static/app/views/onboarding/sdkConfiguration.tsx

```diff
@@ -3,10 +3,11 @@ import * as qs from 'query-string';
 
 import {platfromToIntegrationMap} from 'app/utils/integrationUtil';
 
-import ProjectSetup from './documentationSetup';
+import DocumentationSetup from './documentationSetup';
 import IntegrationSetup from './integrationSetup';
+import OtherSetup from './otherSetup';
 
-type Props = React.ComponentProps<typeof ProjectSetup> &
+type Props = React.ComponentProps<typeof DocumentationSetup> &
   Omit<React.ComponentProps<typeof IntegrationSetup>, 'integrationSlug'>;
```

Comment:
@roggenkemper We should do the intersection type (`&`) of the all the child components to make sure we are typing this correctly:
```
React.ComponentProps<typeof ProjectSetup> & React.ComponentProps<typeof DocumentationSetup> &
Omit<React.ComponentProps<typeof IntegrationSetup>, 'integrationSlug'>;
```

Reply:
good point, added!